grafana/BloatDetails.json

```diff
@@ -204,8 +295,8 @@
       {
         "allValue": null,
         "current": {
-          "text": "Prod",
-          "value": "Prod"
+          "text": "primary",
```

Comment:
Why this text change ?

Reply:
Grafana changed it, it saves whatever your current value is.  Does nothing to the actual dashboard since it uses job label